sc4s_sync.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# echo all the commands
+set -x
+
+REPOORG=splunk
+if [[  $GITHUB_USER && ${GITHUB_USER-x} ]]
+then
+    echo "GITHUB_USER Found"
+else
+    echo "GITHUB_USER Not found"
+    exit 1
+fi
+if [[  $GITHUB_TOKEN && ${GITHUB_TOKEN-x} ]]
+then
+    echo "GITHUB_TOKEN Found"
+else
+    echo "GITHUB_TOKEN Not found"
+    exit 1
+fi
+
+current_tag=$(cat SC4S_matrix.conf | grep -oP '(?<=VERSION=).*')
+echo $current_tag
+
+latest_tag=$(curl -s https://api.github.com/repos/splunk/splunk-connect-for-syslog/releases/latest | grep "tag_name" | cut -d : -f 2)
+new_value=$(echo $latest_tag | rev | cut -c3- | rev)
+new_value=$(echo $new_value | cut -c3-)
+echo $new_value
+sudo apt-get install -y python3 python3.7 python3-pip
```

Comment:
if we are using python3 container, why we have to install python?

Reply:
removed from logic.